maps/away/mining/mining-signal.dmm

```diff
@@ -823,7 +823,7 @@
 /turf/simulated/floor/carpet/blue,
 /area/outpost/abandoned)
 "cF" = (
-/obj/item/weapon/gun/projectile/manualcycle/imprifle,
+/obj/item/weapon/gun/projectile/heavysniper/imprifle
```

Comment:
This line causes a runtime

`## ERROR: Couldn't find atom path specified in map: /obj/item/weapon/gun/projectile/heavysniper/imprifle
/turf/simulated/floor/carpet/blue
[02:00:00] Runtime in reader.dm,351: list index out of bounds
  proc name: parse grid (/dmm_suite/proc/parse_grid)
  src: /dmm_suite (/dmm_suite)
  call stack:
  /dmm_suite (/dmm_suite): parse grid("\n/obj/item/weapon/gun/project...", "cF", 88, 104, 27, 1, null, /list (/list))
  /dmm_suite (/dmm_suite): load map impl(maps/away/mining/mining-signal..., 1, 1, 27, null, null, 1, null, -inf, inf, -inf, inf, /list (/list))
  /dmm_suite (/dmm_suite): load map(maps/away/mining/mining-signal..., 1, 1, null, null, null, 1, null, null, null, null, null, /list (/list))
  Mining - Planetoid (/datum/map_template/ruin/away_site/mining_signal): load new z(1)
  Away Sites Testing (/datum/map/away_sites_testing): build away sites()
  Mapping (/datum/controller/subsystem/mapping): Initialize(716908)
  Mapping (/datum/controller/subsystem/mapping): DoInitialize(716908)
  Master (/datum/controller/master): Initialize(10, 0)`